src/pages/Login/Login.jsx

```diff
@@ -1,7 +1,88 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './Login.scss';
 
 const Login = () => {
-  return <div>로그인 페이지 입니다.</div>;
+  // ID(이메일)
+  const [id, setID] = useState('');
+  const saveUserID = event => {
+    setID(event.target.value);
+  };
+  // PW(비밀번호)
+  const [pw, setPW] = useState('');
+  const saveUserPW = event => {
+    setPW(event.target.value);
+  };
```

Comment:
id 와 pw를 관리하기위한 state를 각각 만들어서 사용하고, 또 그 state를 각자 컨트롤하기위해 서로다른 함수를 만들었습니다. 근데 가만 보면 함수의 형태가 많이 비슷하지 않나요?? 해당 로직을 조금 간편하게 줄일 수 있는 방법을 소개하겠습니다.

우선 id와 pw state를 하나의 객체형태로 다뤄야 합니다.
```suggestion
  const [userInfo,setUserInfo] = useState({id:"",pw:""})
```
이 후 event target에서 태그의 속성값을 객체형태로 가져올 수 있다는 점을 활용해, input tag에 `name` 속성을 부여 하고, 그 값을 가져와 객체에 대입할 수 있습니다.
```suggestion
const [userInfo,setUserInfo] = useState({id:"",pw:""})
const handleUserInfo =(event)=>{
  const {name,value} = event.target
  setUserInfo({...userInfo , [name]:value})
}
```
처음 봤을 때 굉장히 낯설고 어려워 보일수는 있으나 앞으로 굉장히 많이 다루시게 될 내용입니다. 해당 리뷰내용 꼭 한 번 도입해보시고 리뷰에 사용된 문법을 확인 할 수 있는 링크 남겨드릴게요!

[스프레드 연산자](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax)
[계산된 속성명](https://xionwcfm.tistory.com/242)
블로그 보고도 이해 안되시면 꼭 gpt에게라도 물어보고 이해해보세요!